lib/services/collections_service.dart

```diff
@@ -66,7 +66,9 @@ class CollectionsService {
         await setCollectionSyncTime(collection.id, null);
         FileRepository.instance.reloadFiles();
       } else {
-        updatedCollections.add(collection);
+        _cacheCollectionAttributes(collection);
+        final updatedCollection = getcollectionWithDecryptedName(collection);
```

Comment:
Why are we decrypting the name here? This would mean that the DB entry would also have the decrypted name. Wouldn't it be better to let `_cacheCollectionAttributes` have this logic?